spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/registry/PolarisServiceRegistry.java

```diff
@@ -101,7 +117,16 @@ public void register(Registration registration) {
 		instanceRegisterRequest.setMetadata(metadataLocalProperties.getContent());
 		instanceRegisterRequest.setProtocol(polarisDiscoveryProperties.getProtocol());
 		instanceRegisterRequest.setVersion(polarisDiscoveryProperties.getVersion());
+
+		//registerFilterHandlerInstances
+		List<RegisterFilterHandler> registerFilterHandlerInstances = registerFilterHandlerInstances();
 		try {
+			registerFilterHandlerInstances.forEach(registerFilterHandler -> {
+				if (!registerFilterHandler.beforeInvoke(new RegisterFilterHandlerContext(instanceRegisterRequest))) {
```

Comment:
Why not operating instanceRegisterRequest directly but operating RegisterFilterHandlerContext?